src/styles/FileInput.scss

```diff
@@ -0,0 +1,15 @@
+.BrainhubFileInput__wrapper {
```

Comment:
We are using lower case convention. So please change it to `brainhub-file-input__wrapper`. And the same with other classes.